packages/vocabularies-types/utils/generate_types.ts

```diff
@@ -47,6 +47,69 @@ const ANNOTATION_TARGETS = [
     'PropertyValue'
 ];
 
+const KNOWN_DYNAMIC_EXPRESSIONS: Record<string, Record<string, boolean | Record<string, boolean>>> = {
+    UI: {
+        Hidden: true,
+        DeleteHidden: true,
+        UpdateHidden: true,
+        Criticality: true,
+        Emphasized: true,
+        SelectionPresentationVariantType: {
+            Text: true
+        },
+        SelectionVariantType: {
+            Text: true
+        },
+        //ValueCriticality: true,
+        DataFieldAbstract: {
+            Criticality: true
+        },
+        Facet: {
+            Label: true
+        },
+        DataFieldForIntentBasedNavigation: {
+            SemanticObject: true,
+            Action: true,
+            NavigationAvailable: true,
+            RequiresContext: true
+        },
+        HeaderInfoType: {
+            TypeName: true,
+            TypeNamePlural: true,
+            TypeImageUrl: true,
+            ImageUrl: true,
+            Initials: true
+        }
+    },
+    Common: {
```

Comment:
[`@Common.ValueListRelevantQualifiers`](https://github.com/SAP/odata-vocabularies/blob/main/vocabularies/Common.xml#L694:~:text=%3CTerm%20Name=%22-,ValueListRelevantQualifiers,-%22) should be dynamic, too, I think

Reply:
will add it

Reply:
this needs to go into `Common`, not `UI` :)

Reply:
whoops

Reply:
:)